ansible/roles/cartodb.app/tasks/main.yml

```diff
@@ -37,14 +37,43 @@
    args:
      chdir: "{{ cartodb_dir }}"
 
+ - name: Create a common data user
+   command: bundle exec rake cartodb:db:create_user QUOTA_IN_BYTES="{{ cartodb_common_data_user_data_quota }}" SUBDOMAIN="{{ cartodb_common_data_username }}" PASSWORD="{{ cartodb_common_data_user_password }}" EMAIL="{{ cartodb_common_data_user_email }}"
+   become_user: "{{ ansible_ssh_user }}"
+   args:
+     chdir: "{{ cartodb_dir }}"
+   when: cartodb_common_data_local
+
+ - name: Remove table limit for common data user
```

Comment:
I agree that the common data user should not be subject to the same account limits. However, if this task is controlled by the `cartodb_common_data_local` flag, then it will also fail on a provision using upstream cartodb. It is also not strictly necessary for local common data, but is strictly necessary for the data import to succeed, so to me it makes more sense to be controlled by the `cartodb_common_data_import` flag.

Alternatively, rather than use the rake task in this step, you could do a SQL update query using psql, which would remove the dependency on a rake task that doesn't potentially exist.